src/zabapgit_repo_impl.prog.abap

```diff
@@ -401,16 +410,32 @@ CLASS lcl_repo IMPLEMENTATION.
     <ls_return>-file-sha1     = lcl_hash=>sha1( iv_type = gc_type-blob
                                                 iv_data = <ls_return>-file-data ).
 
+    lt_cache = mt_local.
     lt_tadir = lcl_tadir=>read( get_package( ) ).
     LOOP AT lt_tadir ASSIGNING <ls_tadir>.
+
+      ls_item-obj_type = <ls_tadir>-object.
+      ls_item-obj_name = <ls_tadir>-obj_name.
+
+      IF mv_last_serialization IS NOT INITIAL. " Try to fetch from cache
```

Comment:
can this IF be removed? if lt_cache is non empty then just lookup object in cache, if lt_cache is empty just look up anyhow, as lookup in an empty table is fast.

if lt_cache/mt_local is filled then mv_last_serialization will also always be filled?